### ISSUE TITLE
"Fetch" not working on remote branches

### ISSUE THREAD

**Issue by ferreroxc** [eamodio/vscode-gitlens#1309]
I met the same problem described in #1093, which is closed. Therefore, I opened this new issue for further discuss.

- GitLens Version: Gitlens (Insiders) v2021.1.404
- Git Version: git version 2.24.3 (Apple Git-128)
- VSCode Version: 1.52.1
- OS Version: MacOS Catalina 10.15.5

Steps to Reproduce:

1. On Vscode Welcome page, hit "clone repository..."
2. Clone the repository from a remote repository
3. Go to another path; clone the same repository; and push some changes to the repository (make sure the remote master is ahead the local master in VScode)
4. Go back to the VScode window and hit "Fetch" button under "Remotes"->"origin"->"master"
![image](https://user-images.githubusercontent.com/32792279/103725922-99820300-5012-11eb-8cfb-d3f82810b555.png)

After step 4, I expect to see the new commit that I pushed in step 3, but the commits stay the same.

The GitLens log for step 4:
```[2021-01-06 02:36:18:557] [2bf] ViewCommands.fetch — e=BranchNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature):branch(origin/master)):origin/master, undefined
[2021-01-06 02:36:18:558] [2c1] GitService.getOrderedRepositories
[2021-01-06 02:36:18:558] [2c2] GitService.getRepositories
[2021-01-06 02:36:18:558] [2c2] GitService.getRepositories completed • 0 ms
[2021-01-06 02:36:18:558] [2c1] GitService.getOrderedRepositories completed • 0 ms
[2021-01-06 02:36:18:558] [2c3] Repository(/users/xc/workspace/work/debug/test_pull_feature/).fetch — {"branch":{"repoPath":"/Users/xc/workspace/work/debug/test_pull_feature","name":"origin/master","remote":true,"current":false,"date":"2021-01-04T10:07:30.000Z","sha":"3f427fe93b1011ad63b6631170c429723874f51e","refType":"branch","id":"/Users/xc/workspace/work/debug/test_pull_feature|remotes/origin/master","detached":false,"state":{"ahead":0,"behind":0}}}
[2021-01-06 02:36:18:560] [2c4] GitService.fetch — e=/Users/xc/workspace/work/debug/test_pull_feature, t={"branch":{"repoPath":"/Users/xc/workspace/work/debug/test_pull_feature","name":"origin/master","remote":true,"current":false,"date":"2021-01-04T10:07:30.000Z","sha":"3f427fe93b1011ad63b6631170c429723874f51e","refType":"branch","id":"/Users/xc/workspace/work/debug/test_pull_feature|remotes/origin/master","detached":false,"state":{"ahead":0,"behind":0}}}
[2021-01-06 02:36:18:560] [2c5] GitService.getRepository — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:561] [2c5] GitService.getRepository returned /Users/xc/workspace/work/debug/test_pull_feature • 0 ms
[2021-01-06 02:36:18:561] [2c6] GitService.getBranches — e=/Users/xc/workspace/work/debug/test_pull_feature, t={}
[2021-01-06 02:36:18:561] [2c6] GitService.getBranches completed • 0 ms
[2021-01-06 02:36:18:561] [2c4] GitService.fetch completed • 0 ms
[2021-01-06 02:36:18:561] [2c7] Repository(/users/xc/workspace/work/debug/test_pull_feature/).fireChange — unknown
[2021-01-06 02:36:18:561] [2c7] Repository(/users/xc/workspace/work/debug/test_pull_feature/).fireChange completed • 0 ms
[2021-01-06 02:36:18:561] [2c0] GitCommandsCommand.execute completed
[2021-01-06 02:36:18:561] [2c3] Repository(/users/xc/workspace/work/debug/test_pull_feature/).fetch completed • 3 ms
[2021-01-06 02:36:18:561] [2bf] ViewCommands.fetch completed • 3 ms
[2021-01-06 02:36:18:811] Repository[test_pull_feature(/users/xc/workspace/work/debug/test_pull_feature/)] firing changes=unknown
[2021-01-06 02:36:18:811] [2c8] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).onRepositoryChanged — e={ repository: test_pull_feature, changes: unknown }
[2021-01-06 02:36:18:811] [2c9] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).triggerChange — true
[2021-01-06 02:36:18:811] [2ca] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).triggerChange — true, false
[2021-01-06 02:36:18:811] [2cb] RemotesViewNode.triggerChange — true, false
[2021-01-06 02:36:18:811] [2cc] RemotesView.refreshNode — e=RemotesViewNode, t=true, i=false
[2021-01-06 02:36:18:811] [2cd] RemotesViewNode.refresh — true
[2021-01-06 02:36:18:811] [2ce] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).dispose
[2021-01-06 02:36:18:811] [2cf] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).unsubscribe
[2021-01-06 02:36:18:812] [2ce] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).dispose completed • 0 ms
[2021-01-06 02:36:18:812] [2cd] RemotesViewNode.refresh completed • 0 ms
[2021-01-06 02:36:18:812] [2c8] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).onRepositoryChanged completed • 0 ms
[2021-01-06 02:36:18:812] [2d0] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).onRepositoryChanged — e={ repository: test_pull_feature, changes: unknown }
[2021-01-06 02:36:18:812] [2d1] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).triggerChange — true
[2021-01-06 02:36:18:812] [2d2] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).triggerChange — true, false
[2021-01-06 02:36:18:812] [2d3] BranchesViewNode.triggerChange — true, false
[2021-01-06 02:36:18:812] [2d4] BranchesView.refreshNode — e=BranchesViewNode, t=true, i=false
[2021-01-06 02:36:18:812] [2d5] BranchesViewNode.refresh — true
[2021-01-06 02:36:18:812] [2d6] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).dispose
[2021-01-06 02:36:18:812] [2d7] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).unsubscribe
[2021-01-06 02:36:18:812] [2d6] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).dispose completed • 0 ms
[2021-01-06 02:36:18:812] [2d5] BranchesViewNode.refresh completed • 0 ms
[2021-01-06 02:36:18:812] [2d0] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).onRepositoryChanged completed • 0 ms
[2021-01-06 02:36:18:812] [2d8] RemotesView.triggerNodeChange — e=RemotesViewNode
[2021-01-06 02:36:18:812] [2d8] RemotesView.triggerNodeChange completed • 0 ms
[2021-01-06 02:36:18:812] [2d9] BranchesView.triggerNodeChange — e=BranchesViewNode
[2021-01-06 02:36:18:812] [2d9] BranchesView.triggerNodeChange completed • 0 ms
[2021-01-06 02:36:18:813] [2cf] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).unsubscribe completed • 1 ms
[2021-01-06 02:36:18:813] [2cc] RemotesView.refreshNode completed • 1 ms
[2021-01-06 02:36:18:813] [2cb] RemotesViewNode.triggerChange completed • 1 ms
[2021-01-06 02:36:18:813] [2d7] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).unsubscribe completed • 1 ms
[2021-01-06 02:36:18:813] [2d4] BranchesView.refreshNode completed • 1 ms
[2021-01-06 02:36:18:813] [2d3] BranchesViewNode.triggerChange completed • 1 ms
[2021-01-06 02:36:18:814] [2ca] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).triggerChange completed • 2 ms
[2021-01-06 02:36:18:814] [2d2] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).triggerChange completed • 1 ms
[2021-01-06 02:36:18:814] [2c9] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).triggerChange completed • 2 ms
[2021-01-06 02:36:18:814] [2d1] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).triggerChange completed • 2 ms
[2021-01-06 02:36:18:816] [2da] GitService.getOrderedRepositories
[2021-01-06 02:36:18:816] [2db] GitService.getRepositories
[2021-01-06 02:36:18:817] [2dc] GitService.getOrderedRepositories
[2021-01-06 02:36:18:817] [2dd] GitService.getRepositories
[2021-01-06 02:36:18:817] [2db] GitService.getRepositories completed • 1 ms
[2021-01-06 02:36:18:817] [2dd] GitService.getRepositories completed • 0 ms
[2021-01-06 02:36:18:817] [2da] GitService.getOrderedRepositories completed • 1 ms
[2021-01-06 02:36:18:817] [2dc] GitService.getOrderedRepositories completed • 0 ms
[2021-01-06 02:36:18:818] [2de] GitService.getBranches — e=/Users/xc/workspace/work/debug/test_pull_feature, t={}
[2021-01-06 02:36:18:822] [2df] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).ensureSubscription
[2021-01-06 02:36:18:822] [2e0] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).subscribe
[2021-01-06 02:36:18:822] [2e0] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).subscribe completed • 0 ms
[2021-01-06 02:36:18:823] [2df] RemotesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).ensureSubscription completed • 0 ms
[2021-01-06 02:36:18:824] [2e1] GitService.getBranches — e=/Users/xc/workspace/work/debug/test_pull_feature, t={"sort":true}
[2021-01-06 02:36:18:825] [/Users/xc/workspace/work/debug/test_pull_feature] git for-each-ref --format=%3ch%3e%(HEAD)%3cn%3e%(refname)%3cu%3e%(upstream:short)%3ct%3e%(upstream:track)%3cr%3e%(objectname)%3cd%3e%(committerdate:iso8601) refs/heads refs/remotes • waiting...
[2021-01-06 02:36:18:842] [/Users/xc/workspace/work/debug/test_pull_feature] git for-each-ref --format=%3ch%3e%(HEAD)%3cn%3e%(refname)%3cu%3e%(upstream:short)%3ct%3e%(upstream:track)%3cr%3e%(objectname)%3cd%3e%(committerdate:iso8601) refs/heads refs/remotes • 24 ms 
[2021-01-06 02:36:18:842] [/Users/xc/workspace/work/debug/test_pull_feature] git for-each-ref --format=%3ch%3e%(HEAD)%3cn%3e%(refname)%3cu%3e%(upstream:short)%3ct%3e%(upstream:track)%3cr%3e%(objectname)%3cd%3e%(committerdate:iso8601) refs/heads refs/remotes • 17 ms (waited) 
[2021-01-06 02:36:18:842] [2e2] GitBranchParser.parse completed • 0 ms
[2021-01-06 02:36:18:842] [2e3] GitService.getRepository — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:843] [2e4] GitBranchParser.parse completed • 0 ms
[2021-01-06 02:36:18:843] [2e5] GitService.getRepository — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:843] [2e3] GitService.getRepository returned /Users/xc/workspace/work/debug/test_pull_feature • 0 ms
[2021-01-06 02:36:18:843] [2e5] GitService.getRepository returned /Users/xc/workspace/work/debug/test_pull_feature • 0 ms
[2021-01-06 02:36:18:843] [2de] GitService.getBranches completed • 25 ms
[2021-01-06 02:36:18:843] [2e6] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).ensureSubscription
[2021-01-06 02:36:18:843] [2e7] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).subscribe
[2021-01-06 02:36:18:843] [2e7] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).subscribe completed • 0 ms
[2021-01-06 02:36:18:843] [2e8] GitService.getBranches — e=/Users/xc/workspace/work/debug/test_pull_feature, t={"sort":{"current":false}}
[2021-01-06 02:36:18:843] [2e1] GitService.getBranches completed • 18 ms
[2021-01-06 02:36:18:843] [2e8] GitService.getBranches completed • 0 ms
[2021-01-06 02:36:18:844] [2e6] BranchesRepositoryNode(gitlens:repository(/Users/xc/workspace/work/debug/test_pull_feature)).ensureSubscription completed • 1 ms
[2021-01-06 02:36:18:844] [2e9] GitService.getRemotes — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:845] [2ea] GitService.getRepository — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:845] [2ea] GitService.getRepository returned /Users/xc/workspace/work/debug/test_pull_feature • 0 ms
[2021-01-06 02:36:18:845] [2e9] GitService.getRemotes completed • 0 ms
[2021-01-06 02:36:18:847] [2eb] GitService.getBranchAheadRange — e=origin/master
[2021-01-06 02:36:18:847] [2ec] GitService.getBranches — e=/Users/xc/workspace/work/debug/test_pull_feature, t={}
[2021-01-06 02:36:18:847] [2ed] GitService.getBranchAheadRange — e=master
[2021-01-06 02:36:18:847] [2ec] GitService.getBranches completed • 0 ms
[2021-01-06 02:36:18:847] [2ed] GitService.getBranchAheadRange completed • 0 ms
[2021-01-06 02:36:18:847] [2ee] GitService.getLog — e=/Users/xc/workspace/work/debug/test_pull_feature, {"limit":10,"ref":"master"}
[2021-01-06 02:36:18:850] [2ef] GitService.getBranchesAndTagsTipsFn — e=/Users/xc/workspace/work/debug/test_pull_feature, t=master
[2021-01-06 02:36:18:850] [2f0] GitService.getBranches — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:850] [2f1] GitService.getTags — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:853] [2f2] GitBranch.getAssociatedPullRequest — e=undefined
[2021-01-06 02:36:18:853] [2eb] GitService.getBranchAheadRange completed • 6 ms
[2021-01-06 02:36:18:853] [2f3] GitService.getLog — e=/Users/xc/workspace/work/debug/test_pull_feature, {"limit":10,"ref":"origin/master"}
[2021-01-06 02:36:18:856] [2f4] GitService.getBranchesAndTagsTipsFn — e=/Users/xc/workspace/work/debug/test_pull_feature, t=origin/master
[2021-01-06 02:36:18:856] [2f5] GitService.getBranches — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:856] [2f6] GitService.getTags — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:857] [/Users/xc/workspace/work/debug/test_pull_feature] git tag -l --format=%3cn%3e%(refname)%3c*r%3e%(*objectname)%3cr%3e%(objectname)%3cd%3e%(creatordate:iso8601)%3cad%3e%(authordate:iso8601)%3cs%3e%(subject) • waiting...
[2021-01-06 02:36:18:857] [2f7] GitBranch.getAssociatedPullRequest — e=undefined
[2021-01-06 02:36:18:857] [2f8] GitService.getRemotes — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:857] [2f9] GitService.getRepository — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:857] [2f0] GitService.getBranches completed • 6 ms
[2021-01-06 02:36:18:857] [2f5] GitService.getBranches completed • 0 ms
[2021-01-06 02:36:18:857] [2f2] GitBranch.getAssociatedPullRequest completed • 4 ms
[2021-01-06 02:36:18:858] [2f9] GitService.getRepository returned /Users/xc/workspace/work/debug/test_pull_feature • 0 ms
[2021-01-06 02:36:18:858] [2f8] GitService.getRemotes completed • 0 ms
[2021-01-06 02:36:18:858] [2f7] GitBranch.getAssociatedPullRequest completed • 1 ms
[2021-01-06 02:36:18:862] [/Users/xc/workspace/work/debug/test_pull_feature] git tag -l --format=%3cn%3e%(refname)%3c*r%3e%(*objectname)%3cr%3e%(objectname)%3cd%3e%(creatordate:iso8601)%3cad%3e%(authordate:iso8601)%3cs%3e%(subject) • 12 ms 
[2021-01-06 02:36:18:862] [/Users/xc/workspace/work/debug/test_pull_feature] git tag -l --format=%3cn%3e%(refname)%3c*r%3e%(*objectname)%3cr%3e%(objectname)%3cd%3e%(creatordate:iso8601)%3cad%3e%(authordate:iso8601)%3cs%3e%(subject) • 5 ms (waited) 
[2021-01-06 02:36:18:863] [2fa] GitTagParser.parse completed • 0 ms
[2021-01-06 02:36:18:863] [2fb] GitService.getRepository — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:863] [2fc] GitTagParser.parse completed • 0 ms
[2021-01-06 02:36:18:863] [2fd] GitService.getRepository — e=/Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:863] [2fb] GitService.getRepository returned /Users/xc/workspace/work/debug/test_pull_feature • 0 ms
[2021-01-06 02:36:18:864] [2fd] GitService.getRepository returned /Users/xc/workspace/work/debug/test_pull_feature • 0 ms
[2021-01-06 02:36:18:864] [2f1] GitService.getTags completed • 13 ms
[2021-01-06 02:36:18:864] [2f6] GitService.getTags completed • 7 ms
[2021-01-06 02:36:18:864] [2ef] GitService.getBranchesAndTagsTipsFn completed • 13 ms
[2021-01-06 02:36:18:864] [2f4] GitService.getBranchesAndTagsTipsFn completed • 7 ms
[2021-01-06 02:36:18:864] [/Users/xc/workspace/work/debug/test_pull_feature] git log --format=%x3c%x2ff%x3e%n%x3cr%x3e%x20%H%n%x3ca%x3e%x20%aN%n%x3ce%x3e%x20%aE%n%x3cd%x3e%x20%at%n%x3cc%x3e%x20%ct%n%x3cp%x3e%x20%P%n%x3cs%x3e%n%B%n%x3c%x2fs%x3e%n%x3cf%x3e --full-history -M -m --name-status -n11 master -- • 16 ms 
[2021-01-06 02:36:18:864] [2fe] GitService.getCurrentUser — /Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:864] [2fe] GitService.getCurrentUser completed • 0 ms
[2021-01-06 02:36:18:864] [2ff] GitLogParser.parse
[2021-01-06 02:36:18:864] [2ff] GitLogParser.parse completed • 0 ms
[2021-01-06 02:36:18:864] [2ee] GitService.getLog completed • 17 ms
[2021-01-06 02:36:18:868] [/Users/xc/workspace/work/debug/test_pull_feature] git log --format=%x3c%x2ff%x3e%n%x3cr%x3e%x20%H%n%x3ca%x3e%x20%aN%n%x3ce%x3e%x20%aE%n%x3cd%x3e%x20%at%n%x3cc%x3e%x20%ct%n%x3cp%x3e%x20%P%n%x3cs%x3e%n%B%n%x3c%x2fs%x3e%n%x3cf%x3e --full-history -M -m --name-status -n11 origin/master -- • 14 ms 
[2021-01-06 02:36:18:868] [300] GitService.getCurrentUser — /Users/xc/workspace/work/debug/test_pull_feature
[2021-01-06 02:36:18:868] [300] GitService.getCurrentUser completed • 0 ms
[2021-01-06 02:36:18:868] [301] GitLogParser.parse
[2021-01-06 02:36:18:868] [301] GitLogParser.parse completed • 0 ms
[2021-01-06 02:36:18:868] [2f3] GitService.getLog completed • 14 ms
```

The GitLens (Git) log for step 4:

```
[2021-01-06 02:35:02:678] [/Users/xc/workspace/work/debug/test_pull_feature] git for-each-ref --format=%3ch%3e%(HEAD)%3cn%3e%(refname)%3cu%3e%(upstream:short)%3ct%3e%(upstream:track)%3cr%3e%(objectname)%3cd%3e%(committerdate:iso8601) refs/heads refs/remotes • 15 ms 
[2021-01-06 02:35:02:678] [/Users/xc/workspace/work/debug/test_pull_feature] git for-each-ref --format=%3ch%3e%(HEAD)%3cn%3e%(refname)%3cu%3e%(upstream:short)%3ct%3e%(upstream:track)%3cr%3e%(objectname)%3cd%3e%(committerdate:iso8601) refs/heads refs/remotes • 9 ms (waited) 
[2021-01-06 02:35:02:698] [/Users/xc/workspace/work/debug/test_pull_feature] git log --format=%x3c%x2ff%x3e%n%x3cr%x3e%x20%H%n%x3ca%x3e%x20%aN%n%x3ce%x3e%x20%aE%n%x3cd%x3e%x20%at%n%x3cc%x3e%x20%ct%n%x3cp%x3e%x20%P%n%x3cs%x3e%n%B%n%x3c%x2fs%x3e%n%x3cf%x3e --full-history -M -m --name-status -n11 master -- • 14 ms 
[2021-01-06 02:35:02:699] [/Users/xc/workspace/work/debug/test_pull_feature] git tag -l --format=%3cn%3e%(refname)%3c*r%3e%(*objectname)%3cr%3e%(objectname)%3cd%3e%(creatordate:iso8601)%3cad%3e%(authordate:iso8601)%3cs%3e%(subject) • 12 ms 
[2021-01-06 02:35:02:699] [/Users/xc/workspace/work/debug/test_pull_feature] git tag -l --format=%3cn%3e%(refname)%3c*r%3e%(*objectname)%3cr%3e%(objectname)%3cd%3e%(creatordate:iso8601)%3cad%3e%(authordate:iso8601)%3cs%3e%(subject) • 5 ms (waited) 
[2021-01-06 02:35:02:703] [/Users/xc/workspace/work/debug/test_pull_feature] git log --format=%x3c%x2ff%x3e%n%x3cr%x3e%x20%H%n%x3ca%x3e%x20%aN%n%x3ce%x3e%x20%aE%n%x3cd%x3e%x20%at%n%x3cc%x3e%x20%ct%n%x3cp%x3e%x20%P%n%x3cs%x3e%n%B%n%x3c%x2fs%x3e%n%x3cf%x3e --full-history -M -m --name-status -n11 origin/master -- • 13 ms
```

When I run `git fetch origin mater` in terminal, the commits view changed to this, as I expected:
![image](https://user-images.githubusercontent.com/32792279/103726450-e9ad9500-5013-11eb-98db-40464211b918.png)

**Comment by eamodio** [comment]
Can you please verify this fix in tomorrow's insiders edition?

You can install the insiders edition from [here](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens-insiders). Be sure to disable/uninstall the stable version of GitLens first.

**Comment by ferreroxc** [comment]
> Can you please verify this fix in tomorrow's insiders edition?
> 
> You can install the insiders edition from [here](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens-insiders). Be sure to disable/uninstall the stable version of GitLens first.

The "fetch" button in Gitlens(Insiders) v2021.1.604 works perfectly fine. Thank you so much!!
